Seed word numbering

smk762 opened the pull request:
resolves https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/629

tonymorony commented:
lgtm, thanks!

@smk762 since you are in this part of code atm - could you please have a look on https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1603 (maybe there is an easy fix)?

smk762 commented:
- Fixed error message persist on back > next
- Set static error message area to avoid dancing
- Refactored random seed word subset selection fuctions

tonymorony reviewed (approved):
works good now
